plat/kvm/include/kvm-arm64/image.h

```diff
@@ -6,7 +6,11 @@
 #ifndef __KVM_ARM64_IMAGE_H__
 #define __KVM_ARM64_IMAGE_H__
 
+#if defined(CONFIG_KVM_VMM_QEMU)
```

Comment:
Now that Simon mentioned it. Maybe we can just go with `#if`?

Reply:
Yes will fix thanks